src/apps/Permissions/Home/EntityRow.js

```diff
@@ -3,32 +3,31 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import uniqBy from 'lodash.uniqby'
 import {
-  TableRow,
   TableCell,
+  TableRow,
   Text,
-  theme,
-  BreakPoint,
+  Viewport,
   breakpoint,
+  theme,
 } from '@aragon/ui'
 import IdentityBadge from '../../../components/IdentityBadge'
 import AppInstanceLabel from '../AppInstanceLabel'
 import ViewDetailsButton from './ViewDetailsButton'
 
 class EntityRow extends React.PureComponent {
   static propTypes = {
+    compactView: PropTypes.bool,
     entity: PropTypes.object.isRequired,
     onOpen: PropTypes.func.isRequired,
     roles: PropTypes.array.isRequired,
   }
-
-  handleClick = () => {
-    this.props.onOpen(this.props.entity.address)
+  static defaultProps = {
+    compactView: false,
```

Comment:
Nitpicking a bit in that case, but I try to make a distinction between the global breakpoint (small / medium / large) and how it impacts the local elements (in that case, the row being in “compact” mode or not). For example, we could decide that the compact mode of this row need to be active at exactly 450px wide, or depends on the presence of another element, or any other reason.

Reply:
I bring this up is because I read this code and think `compact` might be a different breakpoint (maybe at 450px) but if I read small/medium/large I already know what the breakpoint refers to.
I understand your point, but I think making the one-to-one reference (when it applies) would make the code more readable.

Reply:
I moved to `smallView` 👍